packages/DateInput/test/DateInput.spec.js

```diff
@@ -48,6 +48,8 @@ describe("DateInput", () => {
   });
 
   it("should show error state if it cannot parse the typing string", () => {
+    jest.spyOn(console, "warn").mockImplementation(() => {}); // suppress a momentjs warning
+
```

Comment:
will this mock be cleared after all tests run?

Reply:
oh allison, so smart!

Reply:
Eagle-eye
![download](https://user-images.githubusercontent.com/23224777/116623412-0fc8af00-a8fb-11eb-8362-362b348af006.jpg)

Reply:
😹